README.md

```diff
@@ -40,8 +40,8 @@ This optional environment variable can be used to send arguments to `postgres in
 An example of how to use `cibuild` to build and test an image:
 
 ```bash
-$ CI=1 POSTGIS_MAJOR=2.2 POSTGIS_VERSION=2.3.2+dfsg-1~exp1.pgdg80+1 \
-  PG_MAJOR=9.5 VARIANT=slim \
+$ CI=1 POSTGIS_MAJOR=2.4 POSTGIS_VERSION=2.4.3+dfsg-2.pgdg90+1 \
+  PG_MAJOR=10.3 VARIANT=slim \
```

Comment:
There's no `postgresql-10.3-postgis-2.4` package yet, so I think we should leave `PG_MAJOR=10` here.

Reply:
Package names look good now.